app/_layout.tsx

```
@@ -98,25 +99,47 @@ const RootLayoutStack = () => {
 
   const appTheme = user?.settings?.theme || colorScheme;
 
+  const linkingWorkaround = () => {
+    Linking.addEventListener("url", ({ url }) => {
+      const match = url.match(new RegExp(`^trendly-creators://(.*)`));
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)